core/flyout_base.js

```diff
@@ -570,9 +571,31 @@ Blockly.Flyout.prototype.createFlyoutInfo_ = function(parsedContent) {
   return {contents: contents, gaps: gaps};
 };
 
+/**
+ * Gets the flyout definition for the dynamic category.
+ * @param {string} categoryName The name of the dynamic category.
+ * @return {!Array.<!Element>} The array of flyout items.
```

Comment:
I don't think so, since what is returned here comes from the user. We only require that whatever is returned from their callback is an array of elements.